common/scala/src/main/resources/application.conf

```diff
@@ -72,7 +72,7 @@ whisk {
             heartbeat-interval-ms = 10000
             enable-auto-commit = false
             auto-offset-reset = earliest
-            max-poll-interval = 360000
+            max-poll-interval-ms = 900000 // 20 minutes
```

Comment:
May be change it to use duration as a unit. `max-poll-interval = 20 minutes`

Reply:
These values are passed directly down to the kafka-setting which don't support durations per my knowledge.

The configuration here is build to be passed directly to the respective consumer/producer without touching it in our code at all.

Reply:
Did not knew that. So configs as mentioned [here][1] are passed as is. May be we can later add a testcase which uses [AdminClient.describeTopic][2] to assert the expected config

[1]: https://kafka.apache.org/documentation/#newconsumerconfigs
[2]: http://kafka.apache.org/0110/javadoc/index.html?org/apache/kafka/clients/admin/AdminClient.html

Reply:
Very good finding! And: 900,000 ms are 15 minutes :-)